src/ExtractXliff/Program.cs

```diff
@@ -187,9 +222,33 @@ static bool AddNamespace(string[] args, ref int i)
 			return false;
 		}
 
+		static bool AddLocalizationMethod(string[] args, ref int i)
+		{
+			// -m  --method = additional localization method [zero or more]
+			++i;
+			if (i < args.Length)
+			{
+				if (args[i] == "--")
+					return false;
+
+				var match = Regex.Match(args[i], @"^(?<namespace>[a-zA-Z_][a-zA-Z0-9_]*(\.[a-zA-Z_][a-zA-Z0-9_]*)*)\.(?<class>[a-zA-Z_][a-zA-Z0-9_]*)\.(?<methodName>[a-zA-Z_][a-zA-Z0-9_]*)$");
```

Comment:
Future developers will at a quick glance say "Oh of course, use the regular expression to capture the content following the last '.' to method name, the second to the last '.' as class and the content before that as namespace. "
Or maybe they will get a headache.
Maybe a brief comment here or on the method would help

Reply:
Betterized